src/blocks/expandable/frontend.scss

```diff
@@ -23,6 +23,7 @@
 .expandable-expander {
 	background-color: #fff;
 	border: 1px solid;
+	cursor: pointer;
```

Comment:
Broadly I understand the argument here, but to borrow some resources from an old debate about this in the Tailwind GH report: while it's common to give `<button>`s `cursor: pointer;`, this is an anti-pattern. The [CSS spec](https://drafts.csswg.org/css-ui-3/#valdef-cursor-pointer) establishes that `cursor: pointer` indicates an element is a link.

This article https://medium.com/simple-human/buttons-shouldnt-have-a-hand-cursor-b11e99ca374b has some additional justification for this.

Reply:
TIL 🤯 thank you @kadamwhite 🙏